superset/databases/filters.py

```diff
@@ -69,8 +69,6 @@ class DatabaseUploadEnabledFilter(BaseFilter):  # pylint: disable=too-few-public
     def apply(self, query: Query, value: Any) -> Query:
         filtered_query = query.filter(Database.allow_file_upload)
 
-        database_perms = security_manager.user_view_menu_names("database_access")
-        schema_access_databases = can_access_databases("schema_access")
```

Comment:
if I compare this filter to the actual csv file upload validator, the current code checks for 
`security_manager.can_access_database(database)`
and 
schema in `database.get_schema_access_for_file_upload()`